pkg/logs/scheduler/scheduler.go

```diff
@@ -220,6 +222,13 @@ func (s *Scheduler) toSources(config integration.Config) ([]*logsConfig.LogSourc
 				// cfg.Type is not overwritten as tailing a file from a Docker or Kubernetes AD configuration
 				// is explicitly supported (other combinations may be supported later)
 				cfg.Identifier = service.Identifier
+				// We copy service and source name from the parent container if they are not set
```

Comment:
nit: Would be good to add a note about what cases can lead to the service/source name being empty on the container

Reply:
The `configs` slice is coming straight from user inputs, I added comment to reflect that.